src/main/java/io/github/zap/commons/ZapPlugin.java

```diff
@@ -29,41 +34,43 @@ public Injector getInjector() {
     public void onEnable() {
         doTimedCall(
                 this::doEnable,
-                String.format("Enabling %s:", getName()),
+                "Enabling",
```

Comment:
isn't the message by itself kind of strange?

Reply:
well the logger would output like 
[zombies] Enabling